negotiation/evaluate.py

```diff
@@ -48,8 +48,9 @@
         cuda.set_device(args.gpuid[0])
 
     # Load the model.
+    ckpt_file = args.checkpoint_files[0] if isinstance(args.checkpoint_files, list) else args.checkpoint_files
```

Comment:
just use the list argument (ngargs='+') so that we don't have to check if it's a list